moto/iam/models.py

```diff
@@ -2423,6 +2426,12 @@ def delete_policy(self, policy_arn):
 
     def create_access_key(self, user_name=None, status="Active"):
         user = self.get_user(user_name)
+        keys = self.list_access_keys(user_name)
+        if len(keys) == LIMIT_KEYS_PER_USER:
```

Comment:
Just to make this more secure against weird states it would be better to make this check a `>=` in my opinion.

```suggestion
        if len(keys) >= LIMIT_KEYS_PER_USER:
```